tests/Unit/Core/Util/ConfigurationVariablesLoaderTest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Tests\Unit\Core\Util;
+
+use PHPUnit\Framework\TestCase;
+
+class ConfigurationVariablesLoaderTest extends TestCase
+{
+
+    public function testLoadEnvVariables() :void{
```

Comment:
```suggestion
    public function testLoadEnvVariables() :void
    {
```
Is it a work-in-progress?